openfecwebapp/api_caller.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+from openfecwebapp.local_config import api_location
+
+# api urls are singular, public urls are plural
+type_map = {
+    'candidates': 'candidate',
+    'committees': 'committee',
+    'candidate': 'candidate',
+    'committee': 'committee'
+}
```

Comment:
Good call, this is a leftover from when I was loading data in the view. I can remove it.